contrib/cloud.google.com/go/pubsub.v1/pubsub.go

```diff
@@ -63,15 +64,32 @@ func (r *PublishResult) Get(ctx context.Context) (string, error) {
 	return serverID, err
 }
 
+type config struct {
+	serviceName string
+}
+
+// A Option is used to customize spans started by WrapReceiveHandler.
+type Option func(cfg *config)
+
+// WithServiceName sets the service name tag for traces started by WrapReceiveHandler.
+func WithServiceName(serviceName string) Option {
+	return func(cfg *config) {
+		cfg.serviceName = serviceName
+	}
+}
+
 // WrapReceiveHandler returns a receive handler that wraps the supplied handler,
 // extracts any tracing metadata attached to the received message, and starts a
 // receive span.
-func WrapReceiveHandler(s *pubsub.Subscription, f func(context.Context, *pubsub.Message)) func(context.Context, *pubsub.Message) {
+func WrapReceiveHandler(s *pubsub.Subscription, f func(context.Context, *pubsub.Message), opts ...Option) func(context.Context, *pubsub.Message) {
+	var cfg config
+	for _, opt := range opts {
+		opt(&cfg)
+	}
+
```

Comment:
Same situation here with the curly brace and the space. Can we please remove this line?